dev/NavigationView/NavigationView.cpp

```diff
@@ -1759,10 +1759,6 @@ void NavigationView::UpdatePaneButtonsWidths()
 
     const auto newButtonWidths = [this]()
     {
-        if (DisplayMode() == winrt::NavigationViewDisplayMode::Minimal)
-        {
-            return static_cast<double>(c_paneToggleButtonWidth);
-        }
```

Comment:
Sure that removing this won't cause issues with different values á la #1742?

Reply:
Yup, PaneToggleButtonWidth should be whatever `CompactPaneLength` is. Taking a look at your original PR where you added `UpdatePaneButtonsWidths()`, I believe the test cases are all handled through bindings to TemplateSettings now.